Etabs_Adapter/CRUD/Create/SectionProperty.cs

```diff
@@ -73,7 +73,16 @@ private bool CreateObject(ISectionProperty bhSection)
                 return true;
             }
 
-            SetSection(bhSection as dynamic);
+            if (!m_DBSectionsNames.Contains(bhSection.Name) ||  // Does the name not exist in the database or
```

Comment:
As discussed before, will need to name converters here, between BHoM and Etabs standard names, this is at least for all UK Sections.

Some things I have noted (so far) are.

General:
- Etabs uses all capatal casing (ToUpper())
- Etabs does not use spaces (Replace(" ","")
- Etabs does not type out ".0" for no decimals

Then some specifics:
- UB -> UKB
- UC -> UKC
- UPB -> UKPB
- L -> UKA
- PFC -> UKPFC
- CHS -> CHHF
- RHS -> RHHF
- SHS -> SHHF
- TUB -> UKT
- TUC -> UKT

Might want to add some general Convert methods in the adapter to handle this name conversions. All the specifics above holds true for British shapes. For `ArcelorMittal_British` and `ArcelorMittal_BritishHISTAR` the UB/UC/UBP names can stay as they are.

Reply:
Took a gander through the sections in the BHoM database and the equivalent ones in ETABS database, and as it is now we'll be able to push all of or sections (provided that the dimensions are correct, I'm only fixing the name formatting).

Checked so that our database slots into:
BSShapes2006
AISC14
AISC14M
AISC15
AISC15M
ArcelorMittal_British
ArcelorMittal_BritishHISTAR

and added some pre-formatting on the ETABS section names to facilitate those which have spaces and lowercase letters